modules/oe_content_publication/oe_content_publication.info.yml

```diff
@@ -5,6 +5,7 @@ package: OpenEuropa
 type: module
 core: 8.x
 dependencies:
+  - oe_media:oe_media
```

Comment:
Is this a dependency of this module? If it's not we need to remove it.

Reply:
couldnt find any ref to it so probably not